compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/input/internal/CodepointHelpers.kt

```diff
@@ -20,5 +20,4 @@ internal expect fun CharSequence.codePointAt(index: Int): Int
 
 internal expect fun charCount(codePoint: Int): Int
 
-// TODO: It's not used from common, consider to remove
```

Comment:
YT link?

Reply:
I would just keep it in `commonMain` for the sake of reducing time of overall upstreaming

Reply:
https://youtrack.jetbrains.com/issue/CMP-7423/Common-expect-fun-CharSequence.codePointBefore-is-used-only-in-androidMain